lock_windows.go

```diff
@@ -25,13 +25,12 @@ import (
 
 var (
 	kernel32         = windows.NewLazySystemDLL("kernel32.dll")
+	lockFile         = rootDir + "\\googet.lock"
```

Comment:
I'm not sure this works as intended. rootDir is a flag which is evaluated inside main and prior to main would just be an empty string. Since lockFile is a global var, it's presumably assigned this value prior to main... which means I think this will just cat the empty pre-assignment rootDir string to the file name?

Reply:
Just pass lockFile into obtainLock that way main can crate the path.